docs/guides.md

```diff
@@ -544,67 +544,176 @@ secureboot-certs
+
+
+:::warning
+If the VM has any unsigned kernel modules, they will fail to load after enabling Secure Boot.  Furthermore, the distribution will likely restrict other kernel features that are seen as loop holes in Secure Boot (kexec, /dev/mem, etc...).  Please read the Secure Boot documentation from the distribution.
+:::
+
+#### Setup for Windows VMs
+
+Windows VMs do not require extra installation packages because the Windows Loader and kernel are signed by the keys already installed by the `secureboot-certs` script.  Enabling Secure Boot for the VM in XCP-ng enables Secure Boot in the VM UEFI firmware.
+
+:::warning
+If your VMs have any unsigned drivers, they will fail to load after enabling Secure Boot.
+:::
+
 :::tip
-Optionally, the secure boot mode may be toggled in the
-VM/Advanced tab in Xen Orchestra.
+Unlike Windows, Linux is not signed by Microsoft. Therefore it is required to either install, or use a distribution that comes by default with an intermediary boot loader that *is* signed by Microsoft, if booting Linux with Secure Boot.  This boot loader is typically `shim`.
 :::
 
-After booting the VM, UEFI Secure Boot will be
-enabled.
+### Disable Secure Boot for a Guest VM
+
+#### Disable Secure Boot for a Guest VM using XO
+
+Navigate to the *Advanced* tab and use the **Secure boot** toggle to disable Secure Boot. Reboot the VM and Secure Boot will be disabled.
+
+#### Disable Secure Boot for a Guest VM using XCP-ng Center
 
-#### Disable Secure Boot
+Got to the firmware options for the VM and select *UEFI with SecureBoot*.  Reboot the VM and Secure Boot will be disabled.
+
+#### Disable Secure Boot for a Guest VM using `xe`
+
+In the XCP-ng CLI:
 
 ```
-# Disable secure boot for a UEFI VM
+# Disable Secure Boot for the VM
 xe vm-param-set uuid=<vm-uuid> platform:secureboot=false
 ```
 
-:::tip
-Similar to enabling secure boot, secure boot may also be disabled in the
-VM/Advanced tab in Xen Orchestra.
-:::
+Reboot the VM and Secure Boot will be disabled.
 
-After rebooting the VM, UEFI Secure Boot will be
-disabled.
+### Secure Boot and the UEFI Firmware Menu in the Guest
 
-#### Installing the Revocation List
+Disabling **and** enabling Secure Boot from the UEFI firmware menu inside the guest VM is explicitly disallowed on XCP-ng so as to ensure that guest users can not tamper with the Secure Boot policy set by the host administrator.
+
+Changes to the UEFI secure boot state in the UEFI menu will be ignored in favor of the host administrator's configuration. For example, deselecting **Attempt Secure Boot** will not disable Secure Boot on the next boot.
+
+If disabling Secure Boot by removing keys via Custom Mode is attempted in the UEFI firmware menu, an error will display saying **Only Phyisical Presence User could delete NAME_OF_KEY in custom mode! **.  For example, if attempting to remove the **PK**:
```

Comment:
typo: Phyisical

Reply:
Thanks.  I need to enable spell check for `*.md` in `vim`.